tutorials/intermediate/constrainedlayout_guide.py

```diff
@@ -259,16 +276,16 @@ def example_plot(ax, fontsize=12, nodec=False):
 
 ##########################################
 # rcParams
-# --------
+# ========
 #
-# There are five `rcParams` that can be set, either in a script
-# or in the `matplotlibrc` file.  They all have the prefix
-# `figure.constrained_layout`:
+# There are five :ref:`rcParams<matplotlib-rcparams>` that can be set,
+# either in a script or in the `matplotlibrc` file.
+# They all have the prefix ``figure.constrained_layout``:
 #
-# - `use`: Whether to use constrained_layout. Default is False
-# - `w_pad`, `h_pad`    Padding around axes objects.
+# - *use*: Whether to use constrained_layout. Default is False
```

Comment:
I would tend to use double backticks. But don't really care too much here.

Reply:
You're right, one should be consistent. Double backticks probably make it clearer that this is some code that is entered somewhere.